hasura.planx.uk/migrations/1670838102095_add_sanitised_at_columns/up.sql

```diff
@@ -0,0 +1,14 @@
+alter table "public"."lowcal_sessions" add column "sanitised_at" timestamptz null;
+COMMENT ON COLUMN "public"."lowcal_sessions"."sanitised_at" IS E'Tracks if personal data has been sanitised from record';
+
+alter table "public"."session_backups" add column "sanitised_at" timestamptz null;
+COMMENT ON COLUMN "public"."session_backups"."sanitised_at" IS E'Tracks if personal data has been sanitised from record';
+
+alter table "public"."uniform_applications" add column "sanitised_at" timestamptz null;
+COMMENT ON COLUMN "public"."uniform_applications"."sanitised_at" IS E'Tracks if personal data has been sanitised from record';
+
+alter table "public"."bops_applications" add column "sanitised_at" timestamptz null;
+COMMENT ON COLUMN "public"."bops_applications"."sanitised_at" IS E'Tracks if personal data has been sanitised from record';
+
+alter table "public"."reconciliation_requests" add column "sanitised_at" timestamptz null;
+COMMENT ON COLUMN "public"."reconciliation_requests"."sanitised_at" IS E'Tracks if personal data has been sanitised from record';
```

Comment:
I'm not massively keen on this approach - I think we had a foreign key set up on these audit tables from lowcal_sessions.id we could avoid having sanitised_at columns across audit tables.

This would have the added benefit of deleting these audit records 6 months from when the session has expired / been submitted, as opposed to 6 months from when the audit record was created which is not the same thing.

This PR is already bigger than I'd like though so I'd be happy to take this on as a separate task if we think it's worthwhile to tackle.

Reply:
I don't think there's any case against foreign keys! Agree it would help simplify this and mean we don't require adding a `sanitised_at` to each individual audit table.

Reply:
Perfect - I'll look at picking this up as a separate task.